Tasks/AzureCLI/azureclitask.ts

```diff
@@ -0,0 +1,197 @@
+/// <reference path="../../definitions/node.d.ts" />
+/// <reference path="../../definitions/Q.d.ts" />
+/// <reference path="../../definitions/vsts-task-lib.d.ts" />
+
+import path = require('path');
+import tl = require('vsts-task-lib/task');
+import fs = require('fs');
+import util = require('util');
+
+export class azureclitask {
+    public static async runMain() {
+
+        var toolExecutionError = null;
+        try {
+            tl.setResourcePath(path.join( __dirname, 'task.json'));
+
+            var bash = tl.createToolRunner(tl.which('bash', true));
```

Comment:
Initially we will be going forward with Support for Linux users, when to support windows in still in discussion. It will be clear in a day or two.